### ISSUE TITLE
What is this? 

### ISSUE THREAD

**Issue by punkrokk** [TeamMsgExtractor/msg-extractor#356]
https://products.fileformat.com/email/python/msg-extractor/

**Comment by TheElementalOfDestruction** [comment]
I don't know what that is or who made that. It wasn't made by myself or anyone that I am aware of.

I also couldn't tell you when the entry on that site was added, but it seems to be a while ago given the outdated method of directly using Message.

**Comment by punkrokk** [comment]
Yea all good. Seems weird. Hope you are well!

**Comment by TheElementalOfDestruction** [comment]
Yeah, thanks for bringing it to my attention. I'm gonna see if I can get any more information on it.
